lizmap/modules/lizmap/controllers/features.classic.php

```diff
@@ -154,6 +154,13 @@ public function displayExpression()
         // Filter
         $exp_filter = trim($this->param('exp_filter', 'FALSE'));
 
+        // FieldJSON
+        $fieldJSON = json_decode($this->param('fieldJSON', 'null'), true);
```

Comment:
Use try/catch to be sure the `json_decode` succeeds. If not, use an empty PHP array on error for this field